ui/src/timeMachine/components/Vis.tsx

```diff
@@ -164,7 +164,7 @@ const mstp = (state: AppState): StateProps => {
   const fillColumns = getFillColumnsSelection(state)
   const symbolColumns = getSymbolColumnsSelection(state)
 
-  const timeZone = state.app.persisted.timeZone
+  const timeZone = getTimeZone(state)
```

Comment:
so the line this is replacing wouldn't set the timeZone to `local` if it `state.app.persisted.timeZone` didn't exist. this function will. is that okay for this component? i imagine it is, but just want to be explicit

Reply:
I also noticed that. It's weird because we set `local` as the default in only one place, but I think it's good to have a default for this value, especially if it's a required property. I'm very wary of assigning defaultProps, but in this case I think it's safe to assign a default even if none were specified